sphinx/frontend-bundling.rst

```diff
@@ -0,0 +1,20 @@
+Frontend Bundling
+=================
+
+Webpack is being used to generate frontend bundles based on TypeScript code that is compiled to JavaScript as well as SCSS code which is being transpiled to CSS. The following bundles are defined:
+
+1. `editor.js/css`: Contains TinyMCE and related assets. It was decided to bundle TinyMCE in a separate file because it is the largest of all bundled assets
+2. `main.js/css`: Contains everything else
+
+To get started with adding new code go to `src/frontend` and open `index.ts` or `editor.ts` (depending on the bundle that should be changed).
+In this file you can import any TypeScript or JavaScript file or library that is needed for the application.
+Generated assets will be stored in `src/cms/static/generated`. When running `dev-tools/run.sh`, `webpack-dev-server` is being used to generate JavaScript that includes CSS as well as SourceMaps to help debugging the output. For a production build one should run `npm run prod`. This will generate minified js and css files in the same target directory.
```

Comment:
```suggestion
Generated assets will be stored in ``src/cms/static/generated``. When running :github-source:`dev-tools/run.sh`, ``webpack-dev-server`` is being used to generate JavaScript that includes CSS as well as SourceMaps to help debugging the output. For a production build one should run ``npm run prod``. This will generate minified js and css files in the same target directory.
```